apps/emqx_management/priv/emqx_management.schema

```diff
@@ -143,6 +143,10 @@
   {datatype, string}
 ]}.
 
+{mapping, "management.listener.https.key_password", "emqx_management.listeners", [
+  {datatype, string}
```

Comment:
maybe we need to provide a default value ? 
if we upgrade low version, the key is undefined.

Reply:
Also, if it's not defined, it won't be included in the options: https://github.com/emqx/emqx/blob/50ff2edf131b814b9935bde62dcb38a93401cb7e/apps/emqx_management/priv/emqx_management.schema#L185

So, since today there's no way that someone is using a key file with a password for the management listener, it should keep working after the upgrade.  The only problem would be a downgrade, when the user is using a password-protected key file.